runtime/service.go

```diff
@@ -612,12 +617,26 @@ func (s *service) buildVMConfiguration(req *proto.CreateVMRequest) (*firecracker
 		driveBuilder = firecracker.NewDrivesBuilder(defaultRootfsPath)
 	}
 
-	// TODO: Reserve fake drives here (https://github.com/firecracker-microvm/firecracker-containerd/pull/154)
-	// Right now, just a single hardcoded stub drive is allocated
-	driveBuilder = driveBuilder.AddDrive("/dev/null", false, func(drive *models.Drive) {
-		drive.IsRootDevice = firecracker.Bool(false)
-		drive.DriveID = firecracker.String("containerRootfs")
-	})
+	stubDriveIndex := int64(len(driveBuilder.Build()) - 1)
+	containerCount := int(req.ContainerCount)
+	if containerCount == 0 {
```

Comment:
You probably want `< 1` instead of `== 0`

Reply:
Hm, should we be erroring out if they provide invalid values, like negative numbers?

Reply:
That's also reasonable, but not handled in the current code.